src/main/java/team/chisel/compat/Compatibility.java

```diff
@@ -86,6 +88,10 @@ public static void init(FMLPostInitializationEvent event) {
 		if (Loader.isModLoaded("EE3")) {
 			loadEE3Values();
 		}
+		if (Loader.isModLoaded("NotEnoughItems")) {
+			// disable chiseling by scrolling over itemstack
+			GuiInfo.customSlotGuis.add(GuiChisel.class);
```

Comment:
This shouldn't be run on the server side, GuiChisel is client side only.  Please test this on a MP server.